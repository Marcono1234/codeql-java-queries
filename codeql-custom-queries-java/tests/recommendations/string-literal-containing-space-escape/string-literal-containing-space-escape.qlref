./recommendations/string-literal-containing-space-escape.ql
