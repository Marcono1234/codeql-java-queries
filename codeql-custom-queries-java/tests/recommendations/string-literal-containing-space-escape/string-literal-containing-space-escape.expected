| Test.java:3:9:3:20 | "test\\stext" | Contains space escape sequence `\\s` |
| Test.java:4:9:4:20 | "\\s test \\s" | Contains space escape sequence `\\s` |
| Test.java:5:9:5:26 | "\\\\\\s not escaped" | Contains space escape sequence `\\s` |
